### PR TITLE
"Manage Students" fixes for provider-managed sections

### PR DESCRIPTION
**Pull request by joshlory**
This tab still uses the old `/v2/sections` API so we don't have access to attributes like `provider_managed`.  Use the string names for now, until this view is converted to React.

+cc @poorvasingal

### CODE REVIEW

File: apps/src/sites/code.org/pages/public/teacher-dashboard/index.js [hunk 1]
```diff
@@ -259,6 +259,10 @@ function main() {
       }
     );
 
+    $scope.teacher_managed_section = function (section) {
+      return ['email', 'word', 'picture'].indexOf(section.login_type) > -1;
```
Comment (davidsbailey) [inline apps/src/sites/code.org/pages/public/teacher-dashboard/index.js]: nit: could use es6 `Array.prototype.includes` here instead of `indexOf`

Comment (joshlory) [reply apps/src/sites/code.org/pages/public/teacher-dashboard/index.js]: Do you know if this gets polyfilled by our Babel plugin?  I was a little worried about breaking IE11 but I could be totally wrong...

**Comment by joshlory** [comment]
Thanks for the quick review!  Hope you feel better!

File: apps/src/sites/code.org/pages/public/teacher-dashboard/index.js (hunk 1 above)
Comment (davidsbailey) [reply apps/src/sites/code.org/pages/public/teacher-dashboard/index.js]: that's exactly right, this gets polyfilled by babel. ES6 is safe to use anywhere in apps/ and nowhere outside apps.

**Pull request merged**